README.md

```diff
@@ -1,44 +1,48 @@
 ## Workflow for Metagenome annotation                                                                                                                                                                                                                      
-This workflow is based on the JGI/IMG annotation pipeline ([details](https://github.com/kellyrowland/img-omics-wdl)). This is still in progress.  It takes assembled metagenomes and generates structrual and functional annotations.
+This workflow takes assembled metagenomes and generates structural and functional annotations. It is based on the JGI/IMG annotation pipeline (`details <https://github.com/kellyrowland/img-omics-wdl>`_) and uses a number of open-source tools and databases to generate the structural and functional annotations. 
```

Comment:
kellys repo is very old. We can replace this with Marcel's repo which is public (https://code.jgi.doe.gov/img/img-pipelines/img-annotation-pipeline/)  or the publication (https://journals.asm.org/doi/10.1128/msystems.00804-20)